Voycar.Api.Web/Features/Members/Endpoints/Get/Personal/Endpoint.cs

```diff
@@ -0,0 +1,44 @@
+namespace Voycar.Api.Web.Features.Members.Endpoints.Get.Personal;
+
+using Repository;
+using Users.Repository;
+
+
+public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<NotFound>>, Mapper>
+{
+    private readonly IUsers _userRepository;
+    private readonly IMembers _memberRepository;
+
+    public Endpoint(IUsers userRepository, IMembers memberRepository)
+    {
+        this._userRepository = userRepository;
+        this._memberRepository = memberRepository;
+    }
+
+
+    public override void Configure()
+    {
+        this.Get("member/personal");
+        this.Roles("member");
+        this.Summary(summary =>
+        {
+            summary.Summary = "Get personal details of logged in member";
+            summary.Description = "Get the personal details of the member who is logged in according to the request cookie";
+            summary.Responses[200] = "If member is found";
+            summary.Responses[400] = "If the member somehow can't be found";
```

Comment:
Wenn mein Vorschlag unten akzeptiert wird (bezüglich Statuscode Member prüfen), dann würde ich den Text entsprechend anpassen:
```suggestion
            summary.Responses[400] = "If the member somehow can't be found or the user is not a member.";
```